JP-3562: Extend snowball core (#8303)
spacetelescope/jwst: CHANGES.rst

- Update minimum required photutils version to 1.5.0 [#8211]

> Jump
> ----
> - Add parameters to the Jump step to allow the override of the
>   parameters for flagging the saturated cores of snowballs in
>   the next integration. [#8303]
> 
> 

Reviewer: ```suggestion
jump
----

- Add parameters to the Jump step to allow the override of the
  parameters for flagging the saturated cores of snowballs in
  the next integration. [#8303]

```

jump
----

> - Add parameters to the ``jump`` step to allow the override of the
>   parameters for flagging the saturated cores of snowballs in

Reviewer: ```suggestion
- Add parameters that control the flagging of saturated cores of snowballs in
```
